certbot/renewal.py

```diff
@@ -387,15 +391,21 @@ def handle_renewal_request(config):
                            "instead. The renew verb may provide other options "
                            "for selecting certificates to renew in the future.")
 
-    if config.certname:
-        conf_files = [storage.renewal_file_for_certname(config, config.certname)]
-    else:
-        conf_files = storage.renewal_conf_files(config)
+    conf_files = [storage.renewal_file_for_certname(config, config.certname)] if config.certname \
```

Comment:
This change makes the code harder to understand, since it can't fit on one line anyway.